### DISCUSSION TITLE
Change lsp progress position

### DISCUSSION THREAD

**Discussion by MrRoiz** [folke/noice.nvim#443] [Q&A] [answered]
Hey, just a quick question, is is possible to change the position of lsp progress with coordinates or something like that to avoid an overlapping?

I disabled notifications and the messages are now displayed where they used to, but lsp progress didn't move above but it stayed exactly where it was before
![image](https://user-images.githubusercontent.com/49358353/234589934-9f8b0f27-b09e-46b3-8033-8c3ac7f1696a.png)

**Comment by yutkat** [comment, marked as answer]
Normally, it should not overlap, but it is possible to adjust the position with this.

```lua
require("noice").setup({
	views = {
		mini = {
			position = {
				row = -2,
				col = "100%",
			},
		},
	},
})
```

If there is interference with something, you may want to try this as well.
https://github.com/j-hui/fidget.nvim

**Reply by MrRoiz** [reply]
Cool! thanks, and I used to use fidget before, but I like more how looks with noice actually haha

**Reply by yutkat** [reply]
I am also trying to unify to noice.nvim as much as possible, but I still use fidget.nvim for progress because noice.nvim cannot filter null-ls action notifications.
